threescale_api/defaults.py

```diff
@@ -190,6 +194,7 @@ def select(self, predicate, **kwargs) -> List['DefaultResource']:
 
     def select_by(self, **params) -> List['DefaultResource']:
         """Select by params - logical and
+        Usage example: select_by(**{'role': 'admin'})
```

Comment:
likely usage is `select_by(role="admin")`

Reply:
done